plugins/extractit/icon_extractit/util/util.py

```diff
@@ -10,4 +10,5 @@ class Regex:
     SHA1 = r"\b([A-Fa-f0-9]{40})\b"
     SHA256 = r"\b([A-Fa-f0-9]{64})\b"
     SHA512 = r"\b([A-Fa-f0-9]{128})\b"
-    URL = r"(?<![a-zA-Z0-9/])(?:(?:(?:https?|ftp):)?\/\/)?(?:\S+(?::\S*)?@)?(?:(?!(?:10|127)(?:\.\d{1,3}){3})(?!(?:169\.254|192\.168)(?:\.\d{1,3}){2})(?!172\.(?:1[6-9]|2\d|3[0-1])(?:\.\d{1,3}){2})(?:[1-9]\d?|1\d\d|2[01]\d|22[0-3])(?:\.(?:1?\d{1,2}|2[0-4]\d|25[0-5])){2}(?:\.(?:[1-9]\d?|1\d\d|2[0-4]\d|25[0-4]))|(?:(?:[a-z0-9\u00a1-\uffff][a-z0-9\u00a1-\uffff_-]{0,62})?[a-z0-9\u00a1-\uffff]\.)+(?:[a-z\u00a1-\uffff]{2,}\.?))(?::\d{2,5})?(?:[/?#-]\S*)?"
+    URL = r"(https?:\/\/(?:www\.)?[-a-zA-Z0-9@:%._\+~#=]{2,256}\.[a-z]{2,6}\b(?:[-a-zA-Z0-9@:%_\+.~#?&\/\/=]*))"
```

Comment:
Hm, good point. I'm not too sure if we need to cover those cases in this update, @mrinehart-r7 weigh in?

Reply:
I think it's okay to fail on `https:google.com` since that's a truly invalid URL. I just tried matching `www.google.com` and `www.google.com.` and both matched.

Reply:
Ok, I'll take that to mean this can stay the same for now :)